cmd/templ/lspcmd/proxy/server.go

```diff
@@ -344,21 +346,36 @@ func (p *Server) Completion(ctx context.Context, params *lsp.CompletionParams) (
 			if !ok {
 				continue
 			}
-			imp := addImport(doc.Lines, item.Detail)
-			te := lsp.TextEdit{
-				Range: lsp.Range{
-					Start: lsp.Position{Line: uint32(imp.LineIndex), Character: 0},
-					End:   lsp.Position{Line: uint32(imp.LineIndex), Character: 0},
-				},
-				NewText: imp.Text,
+
+			te := handleImportTextEdit(doc, &item, item.Detail)
```

Comment:
```suggestion
			te := handleImportTextEdit(doc, item.Detail)
```

The item isn't actually used by `handleImportTextEdit`.